### PR TITLE
Schedule automated sms

### PR DESCRIPTION
**Pull request by maysakanoni**
**What**

- Schedule sms to be sent every friday morning at 8

**Why?**

- So the first stage of arrears classification is achieved

### CODE REVIEW

**Comment by kjdchapman** [comment]
Could we write an automated test around this? Or am I missing tests that already cover it?

File: schedule.yml [hunk 1]
```diff
@@ -12,3 +12,8 @@
   request_all_precompiled_letter_states:
     class: RequestAllPrecompiledLetterStates
     cron: '0 0 10,15 * * *' # 10am and 3pm utc every day
+  
+  green_in_arrears_messages:
+    class: GreenInArrearsMessages
+    cron: '0 0 8 ? * FRI *' #8am every friday
```
Comment (richrace) [inline schedule.yml]: How did you create this cron format?

Comment (richrace) [reply schedule.yml]: This is the code that was removed: https://github.com/LBHackney-IT/lbh-income-api/commit/576e4c51b4d6abe0a9e7d5b95fa30d1f1b4c3f4e#diff-d21d26fbd1e7c54f91539115ccf5f644

Comment (maysakanoni) [reply schedule.yml]: I've updated this

**Comment by richrace** [comment]
This branch has been merged into https://github.com/LBHackney-IT/lbh-income-api/pull/348 and is no longer required.

**Pull request closed without merging**